certbot/main.py

```diff
@@ -16,6 +16,7 @@
 import certbot
 
 from certbot import account
+from certbot import cert_manager
```

Comment:
In the past we used "cert manager" to refer to @jdkasten's tool including a UI for cert management tasks. I'm just wondering about C-MIP nomenclature for a library of methods that can perform such tasks vs. a UI for invoking them.